core/templates/dev/head/player/PlayerServices.js

```diff
@@ -335,6 +377,7 @@ oppia.factory('oppiaPlayerService', [
             (GLOBALS.CATEGORIES_TO_COLORS[_exploration.category] || 'teal') +
             '_large.png');
           _loadInitialState(successCallback);
+          _loadInitialInformationCardData(successCallback);
```

Comment:
I have removed it. 
But  in edit mode the card will be shown without metadatas right?